### ISSUE TITLE
Plugin does not work in conjunction with NoScript

### ISSUE THREAD

**Issue by SafwatHalaby** [tikservices/html5-video-everywhere#66]
NoScript prevents this plugin from working. I tested this on [YouTube.com](https://www.youtube.com/). Making NoScript as non-strict as possible by unchecking "ABE", "Forbid Audio/Video", "XSS", etc. does not resolve this. The only way to resolves this is to either disable NoScript, or to white-list [YouTube.com](https://www.youtube.com/).

It is worth noting that html5-video-everywhere works well if one disables both NoScript and sets `javascript.enabled` to false in Firefox's `about:config`.

**Comment by ghost** [comment]
Works for me. You must have youtube.com whitelisted, and googlevideo.com as well if you want the video to automatically play rather than requiring more clicks to tell NoScript the video doesn't need to be blocked anymore. Does it not work this way for you?

**Comment by lejenome** [comment]
Does No-Script block other extensions scripts injected to the web pages?
Because the extension does not depends on YT hosted scripts, just two
scripts injected to the web page.

**Comment by SafwatHalaby** [comment]
@lejenome No-Script does not block scripts injected via [Greasemonkey](https://addons.mozilla.org/en-US/firefox/addon/greasemonkey/), even if the page is not white-listed and the settings are maximally strict. So, I believe it should be possible for this plugin to do the same. 

@d9h02f It works when I whitelist, but I don't want to whitelist. This interference should not exist either way.

Note that if this is fixed, browsing Youtube with Javascript off and NoScript enabled would become viable. This is an attractive option for e.g. Tor users.
